doc/proposals/samples.md

```diff
@@ -0,0 +1,135 @@
+
+#### e2e tests for Go Memcached Sample
+
+- I as a Go dev operator user, I'd like to know how to cover the projects with integration tests using the test-framework
+
+**Acceptance Criteria** 
+- The GO project should be covered with a few basic integration tests
+- The tests should be working successfully
+- The project should have a makefile target to call the integration test
+- The project should have an small section over how to test and with links for reference
+- The tests should be done using the default go testing lib and the test-framework provided by SDK
+
+#### Unit tests for Ansible Memcached Sample
+
+- I as a Ansible dev operator user, I'd like to know how to cover the projects with unit tests using molecule
+
+**Acceptance Criteria** 
+- The Ansible project should be covered by tests using [molecule](https://github.com/operator-framework/operator-sdk-samples/tree/master/ansible/memcached-operator/molecule) which by default is scaffold
+- The tests should be working successfully
+- The project should have an small section over how to test and with links for reference
+- The project should have a makefile target to call the tests
+
+#### e2e tests for Ansible Memcached Sample
+
+- I as a Ansible dev operator user, I'd like to how to cover the projects with integration tests using molecule
+
+**Acceptance Criteria** 
+- The Ansible project should have a few integration tests using [molecule](https://github.com/operator-framework/operator-sdk-samples/tree/master/ansible/memcached-operator/molecule) which by default is scaffold
+- The tests should be working successfully
+- The project should have an small section over how to test and with links for reference
+- The project should have a makefile target to call the tests
+
+#### Travis Integration
+
+- I as a maintainer, I'd like to ensure that the samples still working after the changes performed in the PR easily
+- I as a dev operator user, I'd like to know how to call the tests in the CI and integrate them with Travis
+
+**Acceptance Criteria** 
+- The Ansible and Go project should be integrated with Travis
+- All PR's made against the master should trigger th CI 
+- The unit and integration tests should be checked in the CI
+- The CI should fail if one of the tests do not pass
+
+#### Coveralls Integration
+
+- I as a dev operator user, I'd like to know good practices to ensure the quality of my operators projects 
+
+**Acceptance Criteria** 
+- Ansible and Go Memcached samples should be integrated with [Coveralls](https://coveralls.io/)
+- All above stories should be achieved successfully
+
+### Risks and Mitigations
+
+May [Coveralls](https://coveralls.io/) do not work well with [molecule](https://github.com/operator-framework/operator-sdk-samples/tree/master/ansible/memcached-operator/molecule) and then, we can just not integrate it with or find another similar tool.
+
+## Open Questions 
+
+How to do the same for Helm operators? 
```

Comment:
@joelanford have we any doc over how to cover helm operators with tests?

Reply:
Yes in [doc/helm/dev/developer_guide.md](https://github.com/operator-framework/operator-sdk/blob/master/doc/helm/dev/developer_guide.md#testing-a-helm-operator-locally).

Reply:
Hi @estroz,

Thank you for the doc. But it is how to run the operator locally and/or in the cluster. I mean how to test the operator with tests/checks unit and integration tests. Do you know that?

Reply:
I'd start with this: https://github.com/operator-framework/operator-sdk/blob/master/hack/tests/e2e-helm.sh